worlds/osrs/__init__.py

```diff
@@ -166,10 +165,7 @@ def create_regions(self) -> None:
                 entrance.connect(self.region_name_to_data[parsed_outbound])
 
                 item_name = self.region_rows_by_name[parsed_outbound].itemReq
-                if "*" not in outbound_region_name and "*" not in item_name:
-                    entrance.access_rule = lambda state, item_name=item_name: state.has(item_name, self.player)
-                    continue
-
+                entrance.access_rule = lambda state, item_name=item_name.replace("*",""): state.has(item_name.replace("*",""), self.player)
```

Comment:
Wondering why this wasn't changed to set_rule or add_rule as well when everything else was

(Ftr: You can add_rule when there is no rule yet)

Reply:
Well, that's the only issue I found, and it's not actually an issue, so I'll just merge this so we can make RC3 as soon as possible

If you want to change it, it can be a followup PR ofc